src/components/TeachingBubble/TeachingBubble.Props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TeachingBubble } from './TeachingBubble';
+
+/**
+ * Checkbox class interface.
+ */
+export interface ITeachingBubble {
+
+}
+
+/**
+ * TeachingBubble component props.
+ */
+export interface ITeachingBubbleProps extends React.Props<TeachingBubble> {
+  /**
+   * The background color of the TeachingBubble
+   * @default white
+   */
+  backgroundColor?: TeachingBubbleBGColor;
+
+  /**
+   * A title for the Teaching Bubble.
+   */
+  title?: string;
+
+  /**
+   * An Image for the Teaching Bubble.
+   */
+  img?: string;
+
+  /**
+   * A body for the Teaching Bubble.
+   */
+  body?: string;
```

Comment:
But they are looking for much more functionality than a string of text with an inline link. They want custom titles, full width images, multiple button styles etc.

I'm happy to let them just copy and paste code examples into the teachingbubble, but at that point, why create the component at all?